front/app/containers/Admin/reporting/components/ReportBuilderPage/ReportRow/index.tsx

```diff
@@ -49,7 +49,8 @@ const ReportRow = ({ report }: Props) => {
   };
 
   const handleViewReport = () => {
-    clHistory.push(`${reportPath}/viewer`);
+    // clHistory.push(`${reportPath}/viewer`);
```

Comment:
Not sure if we will keep this in as it's not the best user experience when it opens in a new tab as you get a flash of the underlaying page first. It was a request from Yaesul.